src/streams/TopicItem.js

```diff
@@ -45,20 +55,26 @@ export default function TopicItem(props: Props) {
   const backgroundData = useSelector(state => ({
     auth: getAuth(state),
     mute: getMute(state),
+    streams: getStreamsById(state),
+    streamsByName: getStreamsByName(state),
     subscriptions: getSubscriptions(state),
+    unread: getUnread(state),
     ownUser: getOwnUser(state),
     flags: getFlags(state),
   }));
 
+  const stream = backgroundData.streamsByName.get(streamName);
+  invariant(stream !== undefined, 'No stream with provided stream name was found.');
+
   return (
     <Touchable
-      onPress={() => onPress(stream, name)}
+      onPress={() => onPress(streamName, name)}
       onLongPress={() => {
-        showHeaderActionSheet({
+        showTopicActionSheet({
           showActionSheetWithOptions,
           callbacks: { dispatch, _ },
           backgroundData,
-          stream,
+          streamId: stream[0],
```

Comment:
I think you want `stream.stream_id` here :wink:

(And in the other spot like it.)